selfdrive/car/toyota/carstate.py

```diff
@@ -142,7 +147,12 @@ def update(self, cp):
     self.a_ego = float(v_ego_x[1])
     self.standstill = not v_wheel > 0.001
 
-    if self.CP.carFingerprint in TSS2_CAR:
+    if self.CP.carFingerprint in NO_DSU_CAR and self.CP.carFingerprint not in TSS2_CAR: # Vehicle that needs offsetting
+      self.angle_steers = cp.vl["STEER_TORQUE_SENSOR"]['STEER_ANGLE'] - self.offset
+      if not self.isoffset:
+        self.offset = self.angle_steers - cp.vl["STEER_ANGLE_SENSOR"]['STEER_ANGLE'] + cp.vl["STEER_ANGLE_SENSOR"]['STEER_FRACTION']
```

Comment:
btw, this should be:
```
        self.offset = self.angle_steers - cp.vl["STEER_ANGLE_SENSOR"]['STEER_ANGLE'] + cp.vl["STEER_ANGLE_SENSOR"]['STEER_FRACTION']
```

Reply:
You sure? In stock OP, it's
```
self.angle_steers = cp.vl["STEER_ANGLE_SENSOR"]['STEER_ANGLE'] + cp.vl["STEER_ANGLE_SENSOR"]['STEER_FRACTION']
```
https://github.com/commaai/openpilot/blob/devel/selfdrive/car/toyota/carstate.py#L148

All I'm trying to do is take the current 
```
cp.vl["STEER_TORQUE_SENSOR"]['STEER_ANGLE']
```
and subtract it from 
```
cp.vl["STEER_ANGLE_SENSOR"]['STEER_ANGLE'] + cp.vl["STEER_ANGLE_SENSOR"]['STEER_FRACTION']
```
https://github.com/commaai/openpilot/blob/devel/opendbc/toyota_prius_2017_pt_generated.dbc#L250

Reply:
Pushed